### PR TITLE
Added workaround changes for Chrome browser version issue

### PR DESCRIPTION
**Pull request by itsmekumari**
Added workaround changes for Chrome browser version issue.

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java [hunk 1]
```diff
@@ -141,10 +141,13 @@ public static void changeDataTypesInOutputSchema(String jsonFieldList) {
...
-      CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.SMALL_TIMEOUT_SECONDS);
-    WaitHelper.waitForElementToBeHidden(
-      CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
+    WaitHelper.waitForElementToBeNotClickableOptionally(CdfPluginPropertiesLocators.getSchemaButton);
+    // TODO : Remove the commented lines of code once the bug is resolved.
+    // Bug : https://cdap.atlassian.net/browse/PLUGIN-1693
+//    WaitHelper.waitForElementToBeOptionallyDisplayed(
+//      CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.SMALL_TIMEOUT_SECONDS);
+//    WaitHelper.waitForElementToBeHidden(
+//      CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
```
Comment (Vipinofficial11) [inline src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: It looks a bit messy can we keep it this way ? by using multi line comments
```
/*
TODO : Remove the commented lines of code once the bug is resolved by Chrome Team.
 Bug : https://cdap.atlassian.net/browse/PLUGIN-1693 
 */
 
 
 /*
 WaitHelper.waitForElementToBeOptionallyDisplayed(
      CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.SMALL_TIMEOUT_SECONDS);
 WaitHelper.waitForElementToBeHidden(     CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
 */
```

File: src/main/java/io/cdap/e2e/utils/WaitHelper.java [hunk 2]
```diff
@@ -267,4 +267,20 @@ public static boolean waitForTextToBePresentInElementValue(WebElement element, S
                   " with the Default timeout: " + ConstantsUtil.DEFAULT_TIMEOUT_SECONDS + " seconds");
     return SeleniumDriver.getWaitDriver().until(ExpectedConditions.textToBePresentInElementValue(element, text));
   }
+
+  /**
+   * Wait for the element to be click-disabled
```
Comment (Vipinofficial11) [inline src/main/java/io/cdap/e2e/utils/WaitHelper.java]: Wait for the element to be click-disabled to ```Wait for the element to be disabled```

File: src/main/java/io/cdap/e2e/utils/WaitHelper.java [hunk 3]
```diff
@@ -267,4 +267,20 @@ public static boolean waitForTextToBePresentInElementValue(WebElement element, S
...
+   *
+   * @param element WebElement to check whether click is disabled for temporary time
+   */
+  public static boolean waitForElementToBeNotClickableOptionally(WebElement element) {
```
Comment (Vipinofficial11) [inline src/main/java/io/cdap/e2e/utils/WaitHelper.java]: I think we should keep the method name as ```waitForElementToBeDisabled```

File: src/main/java/io/cdap/e2e/utils/WaitHelper.java [hunk 4]
```diff
@@ -267,4 +267,20 @@ public static boolean waitForTextToBePresentInElementValue(WebElement element, S
...
+    logger.info("Waiting for the element: " + element + " not to be clickable ");
```
Comment (Vipinofficial11) [inline src/main/java/io/cdap/e2e/utils/WaitHelper.java]: update to ``` Waiting for the element: " + element + " to be disabled ```

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java (hunk 1 above)
Comment (itsmekumari) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: Added the multi line comments.

File: src/main/java/io/cdap/e2e/utils/WaitHelper.java (hunk 2 above)
Comment (itsmekumari) [reply src/main/java/io/cdap/e2e/utils/WaitHelper.java]: Changed to the suggested text

File: src/main/java/io/cdap/e2e/utils/WaitHelper.java (hunk 3 above)
Comment (itsmekumari) [reply src/main/java/io/cdap/e2e/utils/WaitHelper.java]: Changed to the suggested method name

File: src/main/java/io/cdap/e2e/utils/WaitHelper.java (hunk 4 above)
Comment (itsmekumari) [reply src/main/java/io/cdap/e2e/utils/WaitHelper.java]: Updated to suggested text

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java (hunk 1 above)
Comment (Vipinofficial11) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: LGTM over all, just confused if we should used a single multi line comment or 2 to wrap the bug and code. What do you think ?

Comment (rahuldash171) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: It's good to wrap and comment in a single line . Will change that and raising one Ex PR .

Comment (itsmekumari) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: I think we should have a single multi line comment. Made the changes accordingly.

**Review by Vipinofficial11 — APPROVED** [review]
LGTM

**Pull request merged**